submissions/description/broken_access_control/recommendations.md

```diff
@@ -0,0 +1,3 @@
+# Recommendations
+
+Access controls must reside in a server-side location where a malicious attacker cannot modify the access controls or view their metadata. Access controls should also be thoroughly tested during development and QA phases.
```

Comment:
Unsure if this is really enough information here.
Was unable to provide more advice and keep it generic for this generic overview finding. Please advise.